sql/weapon_skills.sql

```diff
@@ -253,6 +253,7 @@ INSERT INTO `weapon_skills` VALUES (217,'trueflight',0x0000000000000000000001000
 INSERT INTO `weapon_skills` VALUES (218,'leaden_salute',0x00000000000000000000000000000000010000000000,26,0,255,228,2000,16,1,9,1,0,1,34);
 INSERT INTO `weapon_skills` VALUES (220,'wildfire',0x00000000000000000000020000000000020000000000,26,0,2,235,2000,16,1,14,9,0,1,48);
 INSERT INTO `weapon_skills` VALUES (221,'last_stand',0x00000000000100000000010000000000010000000000,26,357,0,241,2000,16,1,11,5,0,1,0);
+INSERT INTO `weapon_skills` VALUES (241,'final_paradise',0x01010101010101010101010101010101010101010101,1,0,0,25,2000,5,1,13,11,0,0,0);
```

Comment:
Shouldn't its ID be 228?

Reply:
I thought it would just be increment. The highest number is 240, but yes, there is no WSes between 226-238. Sorry to ask a stupid question, but how did you figure out the ID number?

Reply:
nvm, I'm an idiot.
Check windower. Find weaponskill IDs:
`    [228] = {id=228,en="Final Paradise",ja="ファイナルパラダイス",element=6,icon_id=590,prefix="/weaponskill",range=2,targets=32},
`

Yeah, lol, at least now I know xD

Reply:
All fixed now.